extensions/wikia/AdEngine/js/provider/evolve2.js

```diff
@@ -17,7 +17,6 @@ define('ext.wikia.adEngine.provider.evolve2', [
 			'320x50,320x100,300x250': 'a',
 			'300x250': 'a'
 		},
-		section = evolveHelper.getSect(),
```

Comment:
Oh, OK. I might've done something wrong while grepping I didn't spot that. Thanks for update!